### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by dlecocq**
I've got a long line of small(ish) PRs ready to go, but I'll group them into logical chunks.

@b4hand @martin-seomoz @seomoz/bigdata

### CODE REVIEW

**Comment by b4hand** [comment]
If the plan is for this to be a dependency of url-py, then it probably needs to start off as open source since url-py is open source.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+Tests are run with the top-level `Makefile`:
+
+```bash
+make test
+```
+
+Contribution
+============
+Refer to the
+[guidelines for Big Data](https://github.com/seomoz/docs/blob/master/bigdata/contributing.md)
```
Comment (b4hand) [inline README.md]: This is internal documentation, and so if the plan is for this to be an open source project, you'll probably need to point somewhere else.

**Comment by dlecocq** [comment]
That's a good point. I'll make it public, and I'll copy out the contribution guidelines into this README.

**Comment by dlecocq** [comment]
Reupped

**Pull request merged**